src/main/java/org/folio/inventory/validation/MarkAsInProcessNonRequestableValidators.java

```diff
@@ -0,0 +1,43 @@
+package org.folio.inventory.validation;
+
+import org.folio.inventory.domain.items.Item;
+import org.folio.inventory.domain.items.ItemStatusName;
+import org.folio.inventory.exceptions.UnprocessableEntityException;
+import org.folio.inventory.support.http.server.ValidationError;
+
+import java.util.EnumSet;
+import java.util.Set;
+import java.util.concurrent.CompletableFuture;
+
+import static java.util.EnumSet.of;
+import static org.folio.inventory.support.CompletableFutures.failedFuture;
+
+public final class MarkAsInProcessNonRequestableValidators {
```

Comment:
These validators all seem to have almost exactly the same code except for the validation message and the acceptable statuses.

Could these classes be consolidated into a single class with a private constructor (that takes the set of statuses and a function that creates the error message) and create the appropriate one in the service method?